packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -245,6 +246,13 @@ function useMediaUploadQueue() {
             };
           }
 
+          if (resource?.blurHash && !resource?.trimData) {
```

Comment:
Why the `trimData` check? 🤔

Reply:
If you are trimming the video, it will have a different poster and different blurhash.

Reply:
Would be good adding a comment with this explanation.

Didn't we originally say that we don't generate a new poster for trimmed videos, because this difference is negligible?